packages/epo-react-lib/.storybook/main.js

```diff
@@ -29,12 +29,19 @@ const config = {
           "@/helpers": path.resolve(__dirname, "../src/helpers"),
           "@/hooks": path.resolve(__dirname, "../src/hooks"),
           "@/layout": path.resolve(__dirname, "../src/layout"),
+          "@/lib": path.resolve(__dirname, "../src/lib"),
           "@/styles": path.resolve(__dirname, "../src/styles"),
           "@/svg": path.resolve(__dirname, "../src/svg"),
           "@/types": path.resolve(__dirname, "../src/types"),
           path: "path-browserify",
         },
       },
+      define: {
+        "process.env.STORYBOOK_ENV": JSON.stringify(process.env.STORYBOOK_ENV),
+        "process.env.STORYBOOK_PUBLIC_BASE_URL": JSON.stringify(
+          process.env.STORYBOOK_PUBLIC_BASE_URL
```

Comment:
pull in the env vars from `import.meta` where Vite bundles them by default and put them into `process.env` where Storybook expects them